File: c/03.loops/perfect_number.c
#include<stdio.h>
int main()
{
	int n,ans;

	printf("enter the number : ");
	scanf("%d",&n);

	ans=0;
	for(int i=1;i<n;i++)
	{
		if(n%i==0)
		{
		ans=ans+i;
		}
	}
	if(ans==n)
	{
	printf("%d is perfect number ", n );
	}
	else
	{
		printf("%d is not pefect number ",n);
	}
	return 0;
}

File: python/1. Fundamentals/perfect_number.py
num=int(input("Enter the number : "))

data=0
for i in range(1,num):
	if num%i==0:
		data=data+i

if data==num:
	print(f'{num} is the perfect number !')
else:
	print(f'{num} is not a perfect number !')
	